.github/workflows/review_cypress_e2e_tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Cypress end-to-end tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - backend/**
+      - client/**
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-16.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          browser: chrome
+          headless: true
+          start: 
+          wait-on: http://localhost:3000
```

Comment:
the baseUrl says https://localhost:3000/ instead of http://localhost:3000/

Reply:
I'm correcting this.